### PR TITLE
Adding degree as a criterion for drawing reduced networks.

### PR DESCRIPTION
**Pull request by sfwolfphys**
As we discussed today, these are the changes made to add degree as a criterion for drawing reduced networks.  I also changed a goofy variable name to something a little more sensible.

### CODE REVIEW

**Comment by sfwolfphys** [comment]
Also, I have changed the name of the `master` branch to `main` on my repo.  I'm not sure that will translate.  If you want to do the same, I've found some [instructions](https://stevenmortimer.com/5-steps-to-change-github-default-branch-from-master-to-main/) that you can use. It was 80% command line, 20% on the github website and took 2 minutes.

I also don't know anything about travis or getting this to CRAN.  I didn't change any documentation.

**Comment by sfwolfphys** [comment]
Ok, so I guess when I update my branch, it automatically updates here.  Cool, no need to manually update the pull request.  Rather than running my own travis build, I'm letting yours do the work.  We shall see if that makes a difference.  I'm not sure what the error is.  It takes a while to get there (builds are testing for 5+ minutes) and the error is rather hidden. :(

**Comment by sfwolfphys** [comment]
[Travis build status](https://travis-ci.com/sfwolfphys/concorR)  Just in case that pending check is still pending...

File: R/CONCOR_blockmodeling.R [hunk 1]
```diff
@@ -13,32 +13,101 @@ make_blk <- function(adj_list, nsplit = 1) {
...
 .edge_dens <- function(adj_mat) {
+  ## Make weighted matrix unweighted
   adj_mat[adj_mat > 0] <- 1
-  a <- sum(adj_mat)
-  m <- length(adj_mat) - sqrt(length(adj_mat))
-  d <- a / m
+
+  a <- sum(adj_mat)   # Count total edges in network
+  m <- length(adj_mat) - sqrt(length(adj_mat))  ## Count possible edges (remove the diagonal)
+  d <- a / m  # actual/possible
```
Comment (atraxler) [inline R/CONCOR_blockmodeling.R:21]: Only change here is to add comments, right? Crazy as I think it is, those are supposed to be stripped out of submitted packages.

**Comment by sfwolfphys** [comment]
LOL. For official package submission, you should exhibit bad coding practices. Fair enough.

On Dec 2, 2020 8:28 AM, Adrienne Traxler <notifications@github.com> wrote:
This email originated from outside ECU.


@atraxler commented on this pull request.

________________________________

In R/CONCOR_blockmodeling.R<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FATraxLab%2FconcorR%2Fpull%2F5%23discussion_r534165745&data=04%7C01%7Cwolfs15%40ecu.edu%7C9e0019ca169f4222805508d896c62587%7C17143cbb385c4c45a36ac65b72e3eae8%7C0%7C0%7C637425125080755499%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=mnTZ1MZj%2Bz%2B6kmWv%2FFxCbRwFDVHYZyDAT0BuddS5RQE%3D&reserved=0>:

> +  ## Make weighted matrix unweighted
   adj_mat[adj_mat > 0] <- 1
-  a <- sum(adj_mat)
-  m <- length(adj_mat) - sqrt(length(adj_mat))
-  d <- a / m
+
+  a <- sum(adj_mat)   # Count total edges in network
+  m <- length(adj_mat) - sqrt(length(adj_mat))  ## Count possible edges (remove the diagonal)
+  d <- a / m  # actual/possible


Only change here is to add comments, right? Crazy as I think it is, those are supposed to be stripped out of submitted packages.

File: R/CONCOR_blockmodeling.R (hunk 1 above)
Comment (sfwolfphys) [reply R/CONCOR_blockmodeling.R]: I just removed the comments in my most recent push.

**Review by atraxler — APPROVED** [review]

**Pull request merged**